receivers/googlechat/v1/config.go

```diff
@@ -13,9 +13,11 @@ import (
 const Version = schema.V1
 
 type Config struct {
-	URL     string `json:"url,omitempty" yaml:"url,omitempty"`
-	Title   string `json:"title,omitempty" yaml:"title,omitempty"`
-	Message string `json:"message,omitempty" yaml:"message,omitempty"`
+	URL             string `json:"url,omitempty" yaml:"url,omitempty"`
+	Title           string `json:"title,omitempty" yaml:"title,omitempty"`
+	Message         string `json:"message,omitempty" yaml:"message,omitempty"`
+	HideOpenButton  bool   `json:"hide_open_button,omitempty" yaml:"hide_open_button,omitempty"`
+	HideVersionInfo bool   `json:"hide_version_info,omitempty" yaml:"hide_version_info,omitempty"`
```

Comment:
Can you please update the schema (IntegrationSchemaVersion) below? This way it will be rendered in UI automatically

Reply:
Sure thing @yuri-tceretian , it's been added